README.md

```diff
@@ -66,17 +71,21 @@
 ### Key Features <a name="key-features"></a>
 
 
-- **[Gitflow]**
-- **[Linters]**
+- **Landing Page for activities**
+- **Mobile first development**
+- **Responsiveness and flexibility for any screen size**
+- **Fast access to social media and other ways of contact**
+- **Easy to the eyes and cultivating at the same time**
+- **Dynamic creation of content** 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
-## 🤯 PROJECT WALKTHROUGH <a name="walk-through"></a>
+## 🤯 Walk through video <a name="walk-through"></a>
 
 
-- [Walk through video](https://www.loom.com/share/20c1077651a044c081ce4eb772794751)
+- [here](https://www.loom.com/share/0d02cb2c31d24d74adf57b188ad122bd)
 
```

Comment:
- [ ] You did a great job at the presentation of the project and you did 😊⤵️

## Highlights 🚀👌
1. Recorded a video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
4. The video contains a demo of the project, showing the pages in different screen sizes.

## Forgotten Things 🍀
- [ ] But I see that you forgot to show some piece of your code as it is from the requirements, so please 😊The video should contain a highlight about a piece of the code the student finds interesting or is proud of making and with that your presentation will be great and professional 👌😊